wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java

```diff
@@ -193,5 +206,17 @@ public WireMockServlet getServlet() {
         + " for response body content and file paths when loading body files. Effectively enables dynamic file loading"
         + " instead of preloading and simplifies defining stubs.")
     boolean globalTransformer() default true;
+
+    @AttributeDefinition(
+      name = "Request journal",
+      description = "Enable the request journal, which records incoming requests for later verification. For long running instances can exhaust the heap."
+    )
+    boolean requestJournalEnabled() default false;
+
+    @AttributeDefinition(
+      name = "Max Request Journal Entries",
+      description = "Set maximum number of entries in request journal (if enabled). When this limit is reached oldest entries will be discarded. 0 means no limits."
+    )
+    int maxRequestJournalEntries() default 0;
```

Comment:
I prefer naming TOPIC\_TRAIT

so requestJournalEnabled and requestJournalMaxSize e.g

Reply:
renamed.
As mentioned before. I prefer to keep it disabled.